src/components/user-interface/visual-editor/visual-editor-components/toolbar-plugin.tsx

```diff
@@ -108,11 +109,20 @@ export const ToolbarPlugin = React.forwardRef<
     (blockType: BLOCK_TYPES) => {
       formatBlock('paragraph');
       if (toolbarState.blockType === blockType) {
-        editor.dispatchCommand(REMOVE_LIST_COMMAND, () => {});
+        editor.dispatchCommand<LexicalCommand<null>, null>(
+          REMOVE_LIST_COMMAND,
+          null
+        );
       } else if (blockType === BLOCK_TYPES.UL) {
-        editor.dispatchCommand(INSERT_UNORDERED_LIST_COMMAND, () => {});
+        editor.dispatchCommand<LexicalCommand<null>, null>(
+          INSERT_UNORDERED_LIST_COMMAND,
+          null
+        );
       } else if (blockType === BLOCK_TYPES.OL) {
-        editor.dispatchCommand(INSERT_ORDERED_LIST_COMMAND, () => {});
+        editor.dispatchCommand<LexicalCommand<null>, null>(
+          INSERT_ORDERED_LIST_COMMAND,
+          null
+        );
```

Comment:
@scottrippey @ryan-roemer To avoid having to pass something that returns `void` I've had to specify `null` as a payload here. I wish inference would work without specifying the type, but I think this is a cleaner middle ground. Yes I tested the functionality to ensure it still toggles.

You can do so by selecting some text and clicking the bullet or number list in the floating toolbar.